### ISSUE TITLE
Multiplication of eval id

### ISSUE THREAD

**Issue by nmddc0211** [vincrichard/EfficientPS#2]
Hey @vincrichard,
thanks a lot for helping me the last time. I am trying to make use of your code in my own project but I stumbled on something that is really confusing me. Why did you multiply the id of thing instances by 1000 to create the canvas?
`
canvas[mask] = instance_train_id_to_eval_id[cls] * 1000 + nb_instance
`

Regards
Dang

**Comment by vincrichard** [comment]
Hello Dang,
As I remember, since it is panoptic segmentation, at the end your prediction must differentiate both the class and the instances. Here I multiply to 1000 because this provide me with the class id being the first 2 digits and the instance id being the remain 3 digits.

I am using panoptic api for the computation of the pq metric. I am creating this id in order to provide panoptic api with the right inputs. See [here](https://github.com/vincrichard/EfficientPS/blob/e8d0548a05f816293a0adb596ec177f174335010/efficientps/panoptic_metrics.py#L44) for the creation of the input.

Relooking back into panoptic api doc, I thing it's just COCO Panoptic dataset standard to do it like that. I did not find where it was clearly said but you may find it by following the link on their repo talking about the COCO panoptic datasets.
[Panoptic api github](https://github.com/cocodataset/panopticapi)

I may have also found that it was like this by debugging the code of the pq metrics, I don't really remember. 

TLDR; It's a way to have both class information and instance number in the case of things classes.

Hope this helps

**Comment by nmddc0211** [comment]
Hey that really helps, thanks a lot!
